src/main/java/name/valery1707/junit/rule/ConditionalIgnoreRule.java

```diff
@@ -2,16 +2,21 @@
 
 import org.junit.Assume;
 import org.junit.rules.MethodRule;
+import org.junit.runners.model.Annotatable;
 import org.junit.runners.model.FrameworkMethod;
 import org.junit.runners.model.Statement;
 
 import javax.annotation.Nonnull;
 import javax.annotation.Nullable;
+import java.lang.annotation.Annotation;
```

Comment:
\[Checkstyle\] WARNING: Wrong lexicographical order for 'java\.lang\.annotation\.Annotation' import\. Should be before 'javax\.annotation\.Nullable'\.